src/sonic_ax_impl/mibs/ietf/rfc2737.py

```diff
@@ -195,29 +160,20 @@ def get_transceiver_description(sfp_type, if_alias):
 
     return "{} for {}".format(sfp_type, if_alias)
 
-def get_transceiver_sensor_description(sensor, if_alias):
+
+def get_transceiver_sensor_description(name, lane_number, if_alias):
     """
-    :param sensor: sensor key name
+    :param name: sensor name
+    :param lane_number: lane number of this sensor
     :param if_alias: interface alias
     :return: description string about sensor
     """
-
-    # assume sensors that is per channel in transceiver port
-    # has digit equals to channel number in the sensor's key name in DB
-    # e.g. rx3power (lane 3)
-    lane_number = list(filter(lambda c: c.isdigit(), sensor))
-
-    if len(lane_number) == 0:
+    if lane_number == 0:
         port_name = if_alias
```

Comment:
>if_alias [](start = 20, length = 8)

Why `lane_number == 0` is treated differently? Should it be `<if_alias>/0` ?

Reply:
lane number is 1 based. 0 means the sensor does not belong to any lane but for the cable.